### ISSUE TITLE
It didnt generate/synthesize any audio

### ISSUE THREAD

**Issue by bouwnak** [SWivid/F5-TTS#934]
### Checks

- [x] This template is only for usage issues encountered.
- [x] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [x] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [x] I am using English to submit this issue to facilitate community communication.

### Environment Details

(venv) PS C:\Users\user\Experiment\F5-TTS> f5-tts_infer-gradio --share
Download Vocos from huggingface charactr/vocos-mel-24khz
C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\vocos\pretrained.py:70: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless 
they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  state_dict = torch.load(model_path, map_location="cpu")

vocab :  C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\f5_tts\infer\examples\vocab.txt
tokenizer :  custom
model :  C:\Users\user\.cache\huggingface\hub\models--SWivid--F5-TTS\snapshots\84e5a410d9cead4de2f847e7c9369a6440bdfaca\F5TTS_Base\model_1200000.safetensors        

vocab :  C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\f5_tts\infer\examples\vocab.txt
tokenizer :  custom
model :  C:\Users\user\.cache\huggingface\hub\models--SWivid--E2-TTS\snapshots\851141880b5ca38050025e98dfdee27dc553f86e\E2TTS_Base\model_1200000.safetensors        

C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\f5_tts\infer\infer_gradio.py:541: UserWarning: You have not specified a value for the `type` parameter. Defaulting to the 'tuples' format for chatbot messages, but this is deprecated and will be removed in a future version of Gradio. Please set type='messages' instead, which uses openai-style dictionaries with 'role' and 'content' keys.
  chatbot_interface = gr.Chatbot(label="Conversation")
Starting app...
* Running on local URL:  http://127.0.0.1:7860
* Running on public URL: https://f6d22210334427510e.gradio.live

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from the terminal in the working directory to deploy to Hugging Face Spaces (https://huggingface.co/spaces)
Device set to use cuda
C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\transformers\models\whisper\generation_whisper.py:573: FutureWarning: The input name `inputs` is deprecated. 
Please make sure to use `input_features` instead.
  warnings.warn(
You have passed task=transcribe, but also have set `forced_decoder_ids` to [[1, None], [2, 50360]] which creates a conflict. `forced_decoder_ids` will be ignored in favor of task=transcribe.
C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\transformers\models\whisper\modeling_whisper.py:550: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:555.)
  attn_output = torch.nn.functional.scaled_dot_product_attention(
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
gen_text 0 Hello I'm vermeil your own personal AI assistant how can I help you today master?
Building prefix dict from the default dictionary ...
Loading model from cache C:\Users\user\AppData\Local\Temp\jieba.cache
Loading model cost 0.727 seconds.
Prefix dict has been built successfully.
C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\gradio\processing_utils.py:749: UserWarning: Trying to convert audio automatically from float32 to 16-bit int format.
  warnings.warn(warning.format(data.dtype))
C:\Users\user\Experiment\F5-TTS\venv\Lib\site-packages\gradio\processing_utils.py:752: RuntimeWarning: invalid value encountered in cast
  data = data.astype(np.int16)

### Steps to Reproduce

the gradio ui local is working properly but when inferencing and audio, it doesn't work

### ✔️ Expected Behavior

It should inference the audio and synthesize it to produce an audio

### ❌ Actual Behavior

This is the result 

![Image](https://github.com/user-attachments/assets/0ed00a85-594b-4541-a28e-39e720bf1591)

![Image](https://github.com/user-attachments/assets/f947c69b-e8f8-46e4-9824-e60915fd1145)

**Comment by SWivid** [comment]
[If the generation output is blank (pure silence), check for ffmpeg installation.
](https://github.com/SWivid/F5-TTS/tree/main/src/f5_tts/infer#inference)
